app/views/pages/cookies.html.erb

```diff
@@ -0,0 +1,161 @@
+        We don't allow Google to use or share our analytics data.
+      </p>
+    </div>
+
+    <p class="govuk-body">
+      Google Analytics sets the following cookies:
+    </p>
+
+    <table class="govuk-table mb1">
+      <thead class="govuk-table__head">
+        <tr>
+          <th class="govuk-table__header">Name</th>
+          <th class="govuk-table__header">Purpose</th>
+          <th class="govuk-table__header">Expires</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell">_ga</td>
+          <td class="govuk-table__cell">This helps us count how many people visit teaching-vacancies.service.gov.uk by tracking if you've visited before</td>
+          <td class="govuk-table__cell">After 2 years</td>
+          </tr>
+          <tr class="govuk-table__row">
+          <td class="govuk-table__cell">_gid</td>
+          <td class="govuk-table__cell">This helps us count how many people visit teaching-vacancies.service.gov.uk by tracking if you've visited before</td>
+          <td class="govuk-table__cell">After 24 hours</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <p class="govuk-body">
+      You can <a class="govuk-link" href="https://tools.google.com/dlpage/gaoptout">opt out of Google Analytics.</a>
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Sessions
+    </h3>
+
+    <p class="govuk-body">
+      A cookie is set to record your session activity.
+    </p>
+
+    <table class='govuk-table mb1'>
+      <thead class='govuk-table__head'>
+        <th class='govuk-table__header'>Name</th>
+        <th class='govuk-table__header'>Purpose</th>
+        <th class='govuk-table__header'>Expires</th>
+      </thead>
+      <tbody>
+        <tr class='govuk-table__row'>
+          <td class='govuk-table__cell'>SESSION COOKIE</td>
+          <td class='govuk-table__cell'>Stores session data</td>
+          <td class='govuk-table__cell'>After 1 hour</td>
+        </tr>
+        </tbody>
+    </table>
+
+    <h3 class='govuk-heading-m'>
+      Our introductory message
+    </h3>
+
+    <p class="govuk-body">
+      You may see a pop-up welcome message when you first visit Teachers: claim back your student loan repayments.
+      We'll store a cookie so that your computer knows you've seen it and knows not to show it again.
+    </p>
+
+    <table class='govuk-table mb1'>
+      <thead class='govuk-table__head'>
+        <th class='govuk-table__header'>Name</th>
+        <th class='govuk-table__header'>Purpose</th>
+        <th class='govuk-table__header'>Expires</th>
+      </thead>
+      <tbody>
+        <tr class='govuk-table__row'>
+          <td class='govuk-table__cell'>SEEN COOKIE</td>
+          <td class='govuk-table__cell'>Saves a message to let us know that you have seen our cookie message</td>
+          <td class='govuk-table__cell'>After 1 month</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <table class='govuk-table mb1'>
```

Comment:
Does this table need a heading and description like the other two above? Or actually, given we don't actually have a survey enabled on the site, should we just get rid of this table?

Reply:
I may be mistaken but I think we need to have a survey for private beta. Jon has mentioned that Rachel will be sorting out a form to handle this.